mint/fakeAPIData.ts

```diff
@@ -284,12 +325,51 @@ export const prevGraph: GraphDefinition = {
     }
 };
 
+export const costGraph: GraphDefinition = {
+    metadata: {
+        x_col: "cases_averted",
+        id_col: "intervention",
+        format: "long",
+        settings: ["irs_use", "net_use"]
+    },
+    series: [
+        {
+            id: "ITN",
+            y_formula: ["({population} / {procure_people_per_net}) * {price_net_standard}  + ({price_delivery} * {population} * (({procure_buffer} + 100)/ 100))"],
+            name: "Pyrethoid ITN",
+            type: "scatter",
+            marker: {color: "blue"}
+        },
+        {
+            id: "IRS + ITN",
+            y_formula: ["({population} / {procure_people_per_net}) * {price_net_standard} + ({price_delivery} * {population} * (({procure_buffer} + 100) / 100)) + 3 * ({price_irs_per_person} * {population})"],
```

Comment:
These formulas don't include the % irs and net coverage? I think I remember seeing a thread to that effect on Teams but it still seems surprising!

Reply:
Yeah, I was suprised too so I confimed with Ellie!  You can see these formulas all worked out and tested against Ellie's algorithm in mintr now: https://github.com/mrc-ide/mintr/pull/18/files#diff-b3dd5a770abfb3b8a5e094ffe2cc2262987d954af3c5a58c6f9011056dc85a9fR35